src/components/admin/Billing/graphs/DataByMonthGraph.tsx

```diff
@@ -165,38 +170,60 @@ function DataByMonthGraph() {
                         color: theme.palette.text.primary,
                         fontWeight: 'normal',
                     },
+                    axisPointer: {
+                        type: 'shadow',
+                    },
                     formatter: (tooltipConfigs: any[]) => {
-                        if (tooltipConfigs.length > 0) {
-                            const config = tooltipConfigs[0];
+                        let content: string | undefined;
 
+                        tooltipConfigs.forEach((config) => {
                             const dataVolume = formatDataVolumeForDisplay(
                                 seriesConfig,
                                 config
                             );
 
-                            const tooltipTitle =
-                                billingHistory
-                                    .map(({ date }) =>
-                                        intl.formatDate(date, {
-                                            month: 'short',
-                                            year: 'numeric',
-                                        })
-                                    )
-                                    .find((date) =>
-                                        date.includes(config.axisValueLabel)
-                                    ) ?? config.axisValueLabel;
-
-                            return `${tooltipTitle}<br />${config.marker} ${dataVolume}`;
-                        } else {
-                            return undefined;
-                        }
+                            if (content) {
+                                content = `${content}
+                                            <div class="tooltipItem">
```

Comment:
Once the styling is moved into `sx` we can share these class names so they are always in sync.